packages/client/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@sphereon/openid4vci-client",
+  "version": "0.3.7-unstable.0",
+  "description": "OpenID for Verifiable Credential Issuance (OpenID4VCI) client",
+  "main": "dist/main/index.js",
+  "types": "dist/main/index.d.ts",
+  "author": "Sphereon",
+  "license": "Apache-2.0",
+  "private": false,
+  "scripts": {
+    "preinstall": "npx only-allow pnpm",
+    "build": "tsc -p ../../tsconfig.build.json",
```

Comment:
Please use build includes instead of linking it in the package.json (see other projects we have)